influxdb.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/influxdb/influxdb/client"
 )
 
+const (
```

Comment:
I feel this could be defined in a better place -- `server.go`, with the other constants there. It makes more sense to group it with those, as opposed to here. `influxdb.go` is in the same package as `server.go` so you can still reference the constant in this file.